creator.html

```diff
@@ -324,18 +318,16 @@ <h5 class="feature-title font-bold mb-1">Frequent Updates</h5>
                             <i class="fa fa-university fa-2x mb-1 deep-purple-text" aria-hidden="true"></i>
                         </div>
                         <div class="col-10 mb-2">
-                            <h5 class="feature-title font-bold mb-1">FREE Forever</h5>
+                            <h5 class="feature-title font-bold mb-1">Free. Forever.</h5>
```

Comment:
```suggestion
                            <h5 class="feature-title font-bold mb-1">FREE. Forever</h5>
```

1. Free has to be capitalized as it's really important
2. It's a title, not need to add a final dot